reference/commands/list.rst

```diff
@@ -1,95 +1,111 @@
 conan list
 ==========
 
-conan list recipes
-------------------
+.. code-block:: bash
+
+    $ conan list -h
+    usage: conan list [-h] [-f FORMAT] [-v [V]] [--logger] [-p PACKAGE_QUERY] [-r REMOTE] [-c] reference
+
+    Lists existing recipes, revisions or packages in the cache or in remotes given a complete
+    reference or a pattern.
+
+    positional arguments:
+      reference             Recipe reference or package reference. Both can contain * as wildcard at any reference field. If
+                            revision is not specified, it is assumed latest one.
+
+    optional arguments:
+      -h, --help            show this help message and exit
+      -f FORMAT, --format FORMAT
+                            Select the output format: json, html
+      -v [V]                Level of detail of the output. Valid options from less verbose to more verbose: -vquiet, -verror,
+                            -vwarning, -vnotice, -vstatus, -v or -vverbose, -vv or -vdebug, -vvv or -vtrace
+      --logger              Show the output with log format, with time, type and message.
+      -p PACKAGE_QUERY, --package-query PACKAGE_QUERY
+                            Only list packages matching a specific query. e.g: os=Windows AND (arch=x86 OR compiler=gcc)
+      -r REMOTE, --remote REMOTE
+                            Remote names. Accepts wildcards
+      -c, --cache           Search in the local cache
+
+
+The ``conan list`` command is an in-depth-oriented command, and it can list recipes and packages
```

Comment:
"in-depth-oriented" in google gave a exclusive depth psychology books and mental health books 🙈 

```suggestion
The ``conan list`` command is an broad search command, and it can list recipes and packages
```

This one is me thinkings WAN... maybe someone has better non-networking word suggestions

```suggestion
The ``conan list`` command is an wide-area command, and it can list recipes and packages
```